modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/assignment/CreateEventForUnassignedUserTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * ******************************************************************************
+ *  * Copyright 2016 Intuit
```

Comment:
thanks shoeffner for the comment. i referenced the tests in the functional-test module for licesing and followed the same. Will change it as specifed above